lib/alsa_midi_device.dart

```diff
@@ -16,6 +16,9 @@ final alsa = a.ALSA(DynamicLibrary.open('libasound.so.2'));
 final int SND_RAWMIDI_STREAM_INPUT = 1;
 final int SND_RAWMIDI_STREAM_OUTPUT = 0;
 
+/// Happens in non-blocking mode when the read buffer is empty
+final int SND_ERROR_EAGAIN = -11; // Resource temporarily unavailable
```

Comment:
This would be better as a static const and I really should have made I really should have made SND_RAWMIDI_STREAM_INPUT and OUTPUT static consts too so I can see why you did that so if you could change all of these that would be great.

Reply:
make them const (not static as they don't below to any class